utils/order-utils/src/signatures.js

```diff
@@ -16,6 +16,10 @@
 
 const ethUtil = require('ethereumjs-util')
 const sigUtil = require('eth-sig-util')
+const web3Eth = require('web3-eth')
+
+const eth = new web3Eth('http://127.0.0.1:8545')
```

Comment:
I'd like to be able to include this module without having an ethereum node running locally. I'm concerned an exception would be thrown in that case. Maybe this could be driven by a env variable so that if the env variable isn't present it doesn't try connecting to a local node?

Reply:
maybe use an infura key in the .env if the network is rinkeby/mainnet then? as we're making contract lookups?